### PR TITLE
Change log next version

### PR DESCRIPTION
**Pull request by bblommers**
Attempt at updating the ChangeLog.
I've gone through all PRs starting at https://github.com/spulec/moto/pull/2305 until now, as I believe that's the first PR after 1.3.13 was released.

I've took some artistic liberty with the format to account for the mere amount of updates - hope that's acceptable :)

### CODE REVIEW

**Comment by spulec** [comment]
Thanks for the great help!

Going to setup a new release shortly.

**Comment by spulec** [comment]
`moto 1.3.14` is now released.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,189 @@
...
+            * list_tags_log_group()
+            * tag_log_group()
+            * untag_log_group()
+        * Config
+            * batch_get_resource_config()
+            * delete_aggregation_authorization()
+            * delete_configuration_aggregator()
+            * describe_aggregation_authorizations()
+            * describe_configuration_aggregators()
+            * describe_identity_pool()
```
Comment (mikegrima) [inline CHANGELOG.md]: This is not a config method.

Comment (bblommers) [reply CHANGELOG.md]: Good spot!
I've raised https://github.com/spulec/moto/pull/2560 to remedy this

**Pull request merged**